packages/core/cli/__tests__/createProject/uncommentProjectOnly.spec.ts

```diff
@@ -0,0 +1,29 @@
+import uncommentProjectOnly from '../../src/scripts/createProject/uncommentProjectOnly';
+
+const fileContent = `
+/* project-only-start
+    ['@vue-storefront/nuxt-theme'],
+    project-only-end */`;
+
+jest.mock('fs', () => ({
+  readFileSync: () => fileContent,
+  writeFileSync: jest.fn()
+}));
+
+import { writeFileSync } from 'fs';
+
+describe('[vsf-next-cli] uncommentProjectOnly', () => {
+  it('removes comments from project on', () => {
```

Comment:
i think you missed something here ;)

Reply:
Good eye! I've just changed it to: `uncomments parts inside "project only" comments`